src/sentry/features/temporary.py

```diff
@@ -422,6 +422,8 @@ def register_temporary_features(manager: FeatureManager):
     manager.add("organizations:transaction-name-normalize", OrganizationFeature, FeatureHandlerStrategy.INTERNAL, default=True, api_expose=False)
     # Enables automatically triggering autofix on issue summary
     manager.add("organizations:trigger-autofix-on-issue-summary", OrganizationFeature, FeatureHandlerStrategy.FLAGPOLE, api_expose=True)
+    # Enables unlimited auto-triggered autofix runs
+    manager.add("projects:unlimited-auto-triggered-autofix-runs", ProjectFeature, FeatureHandlerStrategy.FLAGPOLE, api_expose=False)
```

Comment:
a project flag might become painful. If we're trying to unblock big orgs, they may have hundreds of projects we'll need to flag in